lib/geolocate_service.rb

```diff
@@ -1,4 +1,19 @@
 module GeolocateService
+  # list of supported countries - TODO: need a full list that Google Maps supports
+  COUNTRIES = [
```

Comment:
This is a starting point list and needs to be expanded to all the countries that Google Maps knows about. 🤔 though we can probable get away with just a subset that contain a space (or any other special characters) - as long as `country_name.parameterize == country_name.downcase`, the code should work as it is.

Reply:
There's still an issue with countries for which Google doesn't include a postal code in the geocoded address (I've run into it with South Korea, Iceland, Mongolia, UAE so far, though there's almost certainly others). We'd have to re-work the logic here to address those - probably key off lat/lon instead of postal code.